ast/builtins.go

```diff
@@ -2240,8 +2240,12 @@ var Diff = &Builtin{
  */
 
 var CryptoX509ParseCertificates = &Builtin{
-	Name:        "crypto.x509.parse_certificates",
-	Description: "Returns one or more certificates from the given base64 encoded string containing DER encoded certificates that have been concatenated.",
+	Name: "crypto.x509.parse_certificates",
+	Description: `Returns zero or more certificates from the given encoded string containing
+DER certificate data.
+
+If the input is empty, the function will return null. The input string should be a list of one or more
+concatenated PEM blocks. The whole input of concatenated PEM blocks can optionally be Base64 encoded.`,
```

Comment:
There should be a change in the `builtin_metadata.json` file corresponding to this. Run `make build`.

Reply:
Hmm, that's a good point. I wonder if we should have a CI check for this. 

We have this in the makefile:

```
.PHONY: ci-check-working-copy
ci-check-working-copy: generate
	./build/check-working-copy.sh
```

Should we also make `ci-check-working-copy` run `build` as well as `generate`?

Reply:
I've updated this in 854fa8dc1007c3a42b9dfb8e54f551ff1ebcf99d for now either way.